File: app/src/main/java/skripsi/kohonen/signaturerecognition/MainActivity.java
package skripsi.kohonen.signaturerecognition;

import android.os.Bundle;
import android.speech.tts.TextToSpeech;
import android.support.design.widget.TabLayout;
import android.support.v7.app.AppCompatActivity;

import butterknife.ButterKnife;
import butterknife.InjectView;
import skripsi.kohonen.signaturerecognition.adapter.SectionsPagerAdapter;

public class MainActivity extends AppCompatActivity {
    TabLayout tabs;
    public Database db;
    public Recognition recog;
    public StatusInput statusInput;

    private boolean training = false;

    public boolean ubah = false;
    public int posisiUbah = 0;
    public String hurufUbah = "";
    @InjectView(R.id.container)
    CustomViewPager pager;

    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.inject(this);

        /*pager = (CustomViewPager) findViewById(R.id.container);*/
        pager.setPagingEnabled(false);
//        pager.setSwipeable(true);
        pager.setAdapter(new SectionsPagerAdapter(getSupportFragmentManager()));

        tabs = (TabLayout) findViewById(R.id.tabs);

        tabs.setupWithViewPager(pager);
        tabs.setTabGravity(TabLayout.GRAVITY_FILL);

        db = Database.getInstance(this.getApplicationContext());
        recog = new Recognition(db);

    }
    private TextToSpeech mTts;

}

File: app/src/main/java/skripsi/kohonen/signaturerecognition/MainActivity.kt
package skripsi.kohonen.signaturerecognition

import android.support.design.widget.TabLayout

class MainActivity : AppCompatActivity() {
    var tabs: TabLayout? = null
    var db: Database? = null
    var recog: Recognition? = null
    var statusInput: StatusInput? = null
    private val training = false
    var ubah = false
    var posisiUbah = 0
    var hurufUbah = ""

    @InjectView(R.id.container)
    var pager: CustomViewPager? = null
    protected fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        ButterKnife.inject(this)

        /*pager = (CustomViewPager) findViewById(R.id.container);*/pager!!.isPagingEnabled = false
        //        pager.setSwipeable(true);
        pager.setAdapter(SectionsPagerAdapter(getSupportFragmentManager()))
        tabs = findViewById(R.id.tabs) as TabLayout?
        tabs.setupWithViewPager(pager)
        tabs.setTabGravity(TabLayout.GRAVITY_FILL)
        db = Database.Companion.getInstance(this.getApplicationContext())
        recog = Recognition(db)
    }

    private val mTts: TextToSpeech? = null
}